src/LazyImage/README.md

```diff
@@ -139,3 +139,64 @@ php vendor/bin/phpunit
 cd Resources/assets
 yarn test
 ```
+
+## Want to implement a cache ?
+
+Use symfony/cache
+
+```
+composer require symfony/cache
+```
+
+And create your twig extension :
+
+```php
+<?php
+
+namespace App\Twig;
+
+use Symfony\Component\String\Slugger\AsciiSlugger;
+use Symfony\Contracts\Cache\CacheInterface;
+use Symfony\Contracts\Cache\ItemInterface;
+use Symfony\UX\LazyImage\BlurHash\BlurHashInterface;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
+
+class BlurHashExtension extends AbstractExtension
+{
+    public function __construct(private BlurHashInterface $blurHash, private CacheInterface $cache)
+    {}
```

Comment:
This is PHP 8 code and I don't think we want documentation/examples to be written with.
I think it would be better to write code compatible with the lowest PHP requirement which is [7.2.5](https://github.com/symfony/ux/blob/main/src/LazyImage/composer.json#L29).

WDYT?